adapters/syslog/syslog.go

```diff
@@ -67,7 +77,10 @@ func NewSyslogAdapter(route *router.Route) (router.LogAdapter, error) {
 	default:
 		return nil, errors.New("unsupported syslog format: " + format)
 	}
-	tmpl, err := template.New("syslog").Parse(tmplStr)
+
+	funcMap := template.FuncMap{
+		"getEnvVar":   tplGetEnvVar}
```

Comment:
Would call the function `getenv` more in the line of other default template functions and the normal os.Getenv go function.